docs(how-to): connect to an Elastic Metal server remotely using windows MTA-2197 (#854)
scaleway/docs-content: compute/elastic-metal/how-to/connect-to-server.mdx

You are now connected to your Elastic Metal server.

> ## How to connect from Windows using remote access 

Reviewer: How to connect to a Windows server using remote access

> ## How to connect from Windows using remote access 

Author: ```suggestion
## How to connect to a Windows server using remote access 
```

> To connect to your Elastic Metal server from Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 

Reviewer: To connect to your Elastic Metal server running windows,

> To connect to your Elastic Metal server from Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 

Author: ```suggestion
To connect to your Elastic Metal server running Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 
```

1. [Create an Elastic Metal server](/compute/elastic-metal/how-to/create-server/) and choose a Windows image to run on it.

  <Message type="important">

    - Windows images are not available for the Aluminium range of Elastic Metal servers.

>     - Make sure you save your password securely after creating your Elastic Metal.

Reviewer 2: ```suggestion
    - Make sure you save your password securely after creating your Elastic Metal server.
```

  </Message>

> 2. Download Microsoft Remote Desktop and launch it on your computer. 

Reviewer 2: I'd probably link it again, for convenience?

> 2. Download Microsoft Remote Desktop and launch it on your computer. 

Author: ```suggestion
2. [Download Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12) and launch it on your computer. 
```

3. Click **Add PC**. A configuration page displays. 

  <Lightbox src="scaleway-configuration-page-microsoft-remote.webp" alt="" />

4. Enter your Elastic Metal server's IP address in the **PC name** field and click **Add**. 

5. Right click your newly added PC and select **Connect**.

6. When prompted, enter `Administrateur` in the **Username** field and your password in the **Password** field. 

> 7. Click **Continue**. Your are redirected to the Windows Server Manager.

Author: ```suggestion
7. Click **Continue**. You are redirected to the Windows Server Manager.
```

> You are now connected to your Elastic Metal server from Windows.

Reviewer 2: ```suggestion
You are now connected to your Elastic Metal server running Windows.
```

> You are now connected to your Elastic Metal server from Windows.

Reviewer: You are now connected to your Elastic Metal server running Windows.

> ## How to connect from Windows

Reviewer 2: I think I'd be tempted to have the order of things on this page be:

- How to connect from OSX (Mac) and Linux
- How to connect from Windows
- How to connect to an Elastic Metal server running Windows

...to me it makes a bit more sense that way, but not sure if it's just me?

   <Lightbox src="scaleway-putty-security.webp" alt="" />

   The terminal window displays.

8. Enter the username `root` and press **Enter** to authenticate against the server with your SSH key.
   <Lightbox src="scaleway-putty-logged.webp" alt="" />

You are now connected to your Elastic Metal server.

## How to connect to a Windows server using remote access

To connect to your Elastic Metal server running Windows, you need to download a remote access tool. In this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 

1. [Create an Elastic Metal server](/compute/elastic-metal/how-to/create-server/) and choose a Windows image to run on it.

  <Message type="important">

    - Windows images are not available for the Aluminium range of Elastic Metal servers.
    - Make sure you save your password securely after creating your Elastic Metal server.

  </Message>

2. Download Microsoft Remote Desktop and launch it on your computer.

  <Message type="note">

  - [Download Microsoft Remote Desktop for OSX](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12)

>   - [Download Microsoft Remote Desktop for Linux]()

Author: ```suggestion
```
